tests/test_project_api.py

```diff
@@ -226,9 +229,9 @@ async def test_delete_project(
         debug(datasets)
         assert len(datasets) == 0
 
-        # Check that project-related jobs were deleted
-        stm = select(ApplyWorkflow).join(Project).where(Project.id == p["id"])
-        jobs = await db.execute(stm)
-        jobs = list(jobs)
-        debug(jobs)
-        assert len(jobs) == 0
+        # Assert that total number of jobs is still 1, but without project_id
+        await db.refresh(job)
+        assert not job.project_id
+        assert not job.input_dataset_id
+        assert not job.output_dataset_id
+        assert not job.workflow_id
```

Comment:
I would generally use `something is None` rather than `not something`. It's better defined (what if `workflow_id=0`?) and it's more readable.